rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -3242,6 +3244,9 @@ public void addNodeToken(String nodeUrl, String token) throws RMException {
             registerAndEmitNodeEvent(rmNode.createNodeEvent(RMEventType.NODE_STATE_CHANGED,
                                                             rmNode.getState(),
                                                             rmNode.getProvider().getName()));
+
+            logger.info("The token '" + token + "' is added to the node " + nodeUrl);
```

Comment:
you could add the caller as well in the message

Reply:
Done.